src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubClientTest.java

```diff
@@ -125,4 +132,76 @@ public void verifySaveIntegrationPluginOfType_postsExpectedRequestBody() throws
             fail();
         }
     }
+
+    /**
+     * This test defines the expected request format to be made to SwaggerHub to create an Integration plugin
+     * The values defined as part of the request matcher are what is expected to be set due to the users configuration.
+     * @throws JsonProcessingException
+     */
+    @Test
+    public void verifySaveIntegrationPluginOfType_postsExpectedRequestBodyWithBasePath() throws JsonProcessingException {
+        swaggerHubClient = buildSwaggerHubClient("basePath");
+        //Given
+        String owner = "owner";
+        String api = "api";
+        String version = "1.0.0";
+        String oas = "3.0.0";
+        String branch = "branch-test";
+        boolean enabled = true;
+        String outputFile = "outputFile";
+        String repository = "repository";
+        String repositoryOwner = "repositoryOwner";
+        String scmProvider = "GITHUB";
+        String token = "token";
+        String target = "JSON (Unresolved)";
+        String syncMethod = "Advanced Sync";
+        String name = "Integration Name";
+        String outputFolder = "output/folder";
```

Comment:
There's a test constants class that you could probably use here instead.

Reply:
Have added as constants on top of the class. Will look into creating a constant Test class. Will chat Monday about it. There is a few more we can probably add this also.